client/src/features/live-site/constants/judging.tsx

```diff
@@ -0,0 +1,18 @@
+export function Judging() {
+  return (
+    <>
+      <div className="text-md font-large">
+        <br />
+        - All judging will take place on Discord in the group to which you are assigned.
```

Comment:
Do you want to use `ul` and `li` here?

Reply:
I have no idea, this is my attempt at web dev

Reply:
You probably do, this will give you proper bullet points when rendered.